src/dotnet/APIView/APIViewWeb/Models/UserPreferenceModel.cs

```diff
@@ -13,6 +13,7 @@ public class UserPreferenceModel
         internal IEnumerable<ReviewType> _filterType;
         internal IEnumerable<string> _state;
         internal IEnumerable<string> _status;
+        internal IEnumerable<string> _firstReleaseApproval;
```

Comment:
Do we need enumerable of string here? Should `bool?` be enough to store the flag?

Reply:
Or is it because we allow either option or both approved and pending in UI side?